starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,76 @@
-// Names and Input
+var hacker1 = "Léa";
 
+var hacker2 = "Elin";
 
-//Conditionals
 
+console.log("The driver's name is " + hacker1 + ".");
+console.log("The navigator's name is " + hacker2 + ".");
 
-// Lorem ipsum generator
+if (hacker1.length > hacker2.length) {
+  console.log("The driver has the longest name, it has" + hacker1.length + "characters.");
+} else if (hacker1.length < hacker2.length) {
+  console.log("Yo, navigator has the longest name, it has" + " " + hacker2.length + " " + "characters.");
+} else {
+  console.log("Wow, you both got equally long names, " + hacker1.length + "characters.");
+}
+// end of name and input and conditionals
+
+for (let char in hacker1) {
+  console.log(hacker1[char].toUpperCase() + " ");
```

Comment:
You can do this here to: `splitDriver = () => {
  console.log(hacker1.toLocaleUpperCase().split("").join(" "))
}`